exporter/awsemfexporter/grouped_metric_test.go

```diff
@@ -508,6 +508,69 @@ func TestAddToGroupedMetric(t *testing.T) {
 			}
 		}
 	})
+
+	t.Run("Gauge metric converted to compacted metric of values and counts ", func(t *testing.T) {
+		emfCalcs := setupEmfCalculators()
+		defer require.NoError(t, shutdownEmfCalculators(emfCalcs))
+
+		// Create a gauge metric with multiple data points
+		gpuMetricName := "container_gpu_utilization"
+		gaugeMetric := generateTestGaugeMetricWithDataPoints(gpuMetricName, doubleValueType, []float64{10, 25, 15, 30, 20})
+
+		// Create a config that includes the metric in GaugeMetricsToCompact
+		cfg := createDefaultConfig().(*Config)
+		cfg.GaugeMetricsToCompact = []string{gpuMetricName}
+
+		// Set up the test
+		groupedMetrics := make(map[any]*groupedMetric)
+		rms := gaugeMetric.ResourceMetrics()
+		ilms := rms.At(0).ScopeMetrics()
+		metrics := ilms.At(0).Metrics()
```

Comment:
Can we add a 2nd metric here with the same name and diff datapoints to validate that scenario? 
Per my understanding of current logic, that might log a warning with `Duplicate metric found` and just ignore the values from the 2nd metric.